lib/opentelemetry-mapper.js

```diff
@@ -61,12 +60,13 @@ const SEVERITY_NAME_MAP = {
  *
  * @typedef {Object} MapperOptions
  * @property {string} messageKey
+ * @property {Object.<number, number>} [severityNumberMap]
```

Comment:
Would it make sense to call this severityNumberMapOverrides? Given we already have a constant map with the same name, this name might better describe the purpose of the parameter.

Reply:
I think I'd rather rename the const to something like `DEFAULT_SEVERITY_NAME_MAP`.

I'd actually really prefer to name the option `customLogLevels`, but I'm concerned that may be confusing with how different it is from the pino option of the same name.